docs/sources/datasources/tempo.md

```diff
@@ -182,7 +182,7 @@ A service graph is a visual representation of the relationships between services
 
 To display the service graph:
 
-- [Configure the Grafana Agent](https://grafana.com/docs/tempo/next/grafana-agent/service-graphs/#quickstart) or [Tempo/GET](https://grafana.com/docs/tempo/latest/server_side_metrics/service_graphs/#tempo) to generate service graph data
+- [Configure the Grafana Agent](https://grafana.com/docs/tempo/latest/grafana-agent/service-graphs/#quickstart) or [Tempo/GET](https://grafana.com/docs/tempo/latest/server_side_metrics/service_graphs/#tempo) to generate service graph data
```

Comment:
Hey Kim, all looks great -- just one thing from my short time on Agent docs -- we don't use an article in front of agent -- so it's just Configure Grafana Agent.